shared/services/state/utils.go

```diff
@@ -76,3 +76,59 @@ func GetPendingRPLRewards(cfg *config.RocketPoolConfig, index uint64, rp *rocket
 
 	return rewards.GetPendingRPLRewards(rp, opts)
 }
+
+func msb(n *big.Int) uint {
+	copyN := big.NewInt(0).Set(n)
+	var out uint
+	for copyN.Cmp(big.NewInt(0)) > 0 {
+		copyN.Rsh(copyN, 1)
+		out++
+	}
+
+	// 0-index
+	return out - 1
```

Comment:
1. I would prefer a longer name for the function. 

2. Won't this have an issue if n == 0, overflowing the uint return (-1)?

Reply:
It isn't exported (only usable within `package state`), but i can add a comment/rename.

As for 2- yes, but we never call it with an input less than 2 eth. If we want a general purpose function, the correct index of the msb for 0x00 is -1 anyway (since there is no most significant bit, and index 0 is the lsb)